src/vivarium_gates_mncnh/plugins/time.py

```diff
@@ -28,9 +28,13 @@ def setup(self, builder: Builder) -> None:
             SIMULATION_EVENT_NAMES.NEONATAL,
         )
         self.step_index = 0
-        self.step_name = self.simulation_events[self.step_index]
+        self.step_name = self.simulation_events[self.step_index + 1]
 
     def step_forward(self, index: pd.Index) -> None:
         super().step_forward(index)
-        if self.step_index < len(self.simulation_events) - 1:
+        if self.step_index < len(self.simulation_events) - 2:
```

Comment:
Doesn't this mean that we will repeat the neonatal step if the user specified too many time-steps? I think we should error during `setup` if there will be more than 3 time-steps to protect against this.

Reply:
I actually haven't tested that but I do think we should error.